### DISCUSSION TITLE
Need to access suggestions programatically

### DISCUSSION THREAD

**Discussion by NikhilSaini38** [argenos/nldates-obsidian#69] [Q&A]
I am trying to implement nldates suggestions in quickadd suggester via user scripts. is it possible to expose nldates suggester in API?

**Comment by liamcain** [comment]
There's some information about how to access the natural language parser programmatically here: https://github.com/argenos/nldates-obsidian#for-developers. It won't generate the suggestion list, but you can probably replicate the list with something like this: https://github.com/argenos/nldates-obsidian/blob/fd06ef1646eb8e29f1887e872248549dc27cb5af/src/suggest/date-suggest.ts#L49
